open-metadata-implementation/adapters/open-connectors/configuration-store-connectors/configuration-file-store-connector/src/main/java/org/odpi/openmetadata/adapters/adminservices/configurationstore/file/FileBasedUIServerConfigStoreConnector.java

```diff
@@ -0,0 +1,146 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+package org.odpi.openmetadata.adapters.adminservices.configurationstore.file;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.apache.commons.io.FileUtils;
+import org.odpi.openmetadata.frameworks.connectors.properties.ConnectionProperties;
+import org.odpi.openmetadata.frameworks.connectors.properties.EndpointProperties;
+import org.odpi.openmetadata.userinterface.adminservices.configuration.properties.UIServerConfig;
+import org.odpi.openmetadata.userinterface.adminservices.store.UIServerConfigStoreConnectorBase;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.File;
+import java.io.IOException;
+
+public class FileBasedUIServerConfigStoreConnector extends UIServerConfigStoreConnectorBase
+{
```

Comment:
I thoght we were going to go for a single connector that covered both the sever and the UI Config files?

Reply:
https://github.com/odpi/egeria/pull/1677

Reply:
I've made the connectors generics so they can be reused for convert any configuration. Please see #1677 and reuse the same connectors.